pkg/getty/etc/profile

```diff
@@ -0,0 +1,13 @@
+export CHARSET=UTF-8
+export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+export PAGER=less
+export PS1='(getty ns):\w\$ '
+export PS1='\h:\w\$ '
```

Comment:
This is overwriting your carefully crafted `$PS1` from above.

Why not use a file in `/etc/profile.d/*.sh ` instead of duplicating the whole file?

Also I'm not sure about dropping `\h` since knowing the host is still useful in the getty container, so maybe your snipped should be `export PS1=(getty ns) $PS1"` to just extend the basic one?